### DISCUSSION TITLE
Dietpi-Software | Zigbee2mqtt

### DISCUSSION THREAD

**Discussion by id4vip** [MichaIng/DietPi#5577] [Software Request]
I want to request Dietpi administrators to add zigbee2mqtt in standard dietpi installation software, as it is one of the required
software in home Assistant software for zigbee to MQTT integration.
https://www.zigbee2mqtt.io/

thanks

**Comment by seniorm0ment** [comment]
Really want to see this. Would be so nice to be able to one click install it.

**Comment by id4vip** [comment]
In my setup for running zigbee2mqtt on dietpi user,  i have to give this command so that dietpi user can communicate with coordinator "sudo usermod -G dialout dietpi"
